scripts/bridge/setup/setup.go

```diff
@@ -37,9 +38,11 @@ type DeployResult struct {
 	Error              error
 }
 
-func Deploy(privateKey *string, accountID *string, adminKey *string, network *string, members *int, topicThreshold *uint) DeployResult {
+func Deploy(privateKey *string, accountID *string, adminKey *string, network *string, members *int, cachedPrivateKeys []hedera.PrivateKey, topicThreshold *uint) DeployResult {
 	result := DeployResult{}
 
+	fmt.Printf("%v", cachedPrivateKeys)
```

Comment:
nit: If it's not leftover add some more info

Reply:
forgot to delete it